app/dawn-hub/Views/OnboardingViews/Step4_ChoosePlanView/ChoosePlanView.jsx

```diff
@@ -325,11 +327,12 @@ class ChoosePlanView extends React.Component {
 									and move them to Step 5 if signed in
 								2. DONE If user is signed out, clicking this should take them to Step 4b
 							*/}
+							{/* May have to change the below links depending on GH-2248 */}
 							{loggedIn && (
-								<a className="ChoosePlanView__searchCTAButton" href={plusCheckoutLink} target="_blank" rel="noreferrer">{t('ghostery_dawn_onboarding_start_trial')}</a>
+								<a className="ChoosePlanView__searchCTAButton" href={plusCheckoutLink} target="_blank" rel="noreferrer" onClick={() => { setSetupStep({ setup_step: CHOOSE_PLAN, setup_number: 2, origin: ONBOARDING }); history.push('/onboarding/5'); }}>{t('ghostery_dawn_onboarding_start_trial')}</a>
 							)}
 							{!loggedIn && (
-								<div className="ChoosePlanView__searchCTAButton" onClick={() => next()}>{t('ghostery_dawn_onboarding_start_trial')}</div>
+								<div className="ChoosePlanView__searchCTAButton" onClick={() => { setSetupStep({ setup_step: CHOOSE_PLAN, setup_number: 2, origin: ONBOARDING }); history.push('/onboarding/5'); }}>{t('ghostery_dawn_onboarding_start_trial')}</div>
```

Comment:
Let's not inline handlers that are not one-liners. Besides, looks like we can use the same handler in both places (unless the `setup_number` actually needs to be something other than 2 in one of these cases?)

(also could we constantify the `setup_number` in the handler)

Reply:
Yup, that's way cleaner. I made a `setSetupStepAndMoveToSuccessView` function that takes in `setup_number` constant

Reply:
Great, thank ya